### PR TITLE
Fix catalog image push after rebase

### PR DESCRIPTION
**Pull request by jianrongzhang89**
Quick fix for catalog image push error after the code was rebased on main.

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -3,18 +3,20 @@ module github.com/mongodb/mongodb-atlas-kubernetes
 go 1.16
 
 require (
+	cloud.google.com/go v0.99.0 // indirect
```
Comment (tchughesiv) [inline go.mod:6]: are changes to this file necessary?

File: go.sum [hunk 2]
```diff
@@ -18,8 +18,16 @@ cloud.google.com/go v0.72.0/go.mod h1:M+5Vjvlc2wnp6tjzE102Dw08nGShTscUx2nZMufOKP
...
 cloud.google.com/go v0.81.0/go.mod h1:mk/AM35KwGk/Nm2YSeZbxXdrNK3KZOYHmLkOqC2V6E0=
+cloud.google.com/go v0.83.0/go.mod h1:Z7MJUsANfY0pYPdw0lbnivPx4/vhy/e2FEkSkF7vAVY=
+cloud.google.com/go v0.84.0/go.mod h1:RazrYuxIK6Kb7YrzzhPoLmCVzl7Sup4NrbKPg8KHSUM=
+cloud.google.com/go v0.87.0/go.mod h1:TpDYlFy7vuLzZMMZ+B6iRiELaY7z/gJPaqbMx6mlWcY=
+cloud.google.com/go v0.90.0/go.mod h1:kRX0mNRHe0e2rC6oNakvwQqzyDmg57xJ+SZU1eT2aDQ=
+cloud.google.com/go v0.93.3/go.mod h1:8utlLll2EF5XMAV15woO4lSbWQlk8rer9aLOfLh7+YI=
+cloud.google.com/go v0.94.1/go.mod h1:qAlAugsXlC+JWO+Bke5vCtc9ONxjQT3drlTTnAplMW4=
+cloud.google.com/go v0.97.0/go.mod h1:GF7l59pYBVlXQIBLx3a761cZ41F9bBH3JUlihCt2Udc=
+cloud.google.com/go v0.99.0 h1:y/cM2iqGgGi5D5DQZl6D9STN/3dR/Vx5Mp8s752oJTY=
+cloud.google.com/go v0.99.0/go.mod h1:w0Xx2nLzqWJPuozYQX+hFfCSI8WioryfRDzkoI/Y2ZA=
```
Comment (tchughesiv) [inline go.sum:30]: are changes to this file necessary?

File: pkg/controller/atlasinstance/atlasinstance_test.go [hunk 3]
```diff
@@ -24,16 +24,16 @@ import (
...
 	"github.com/fgrosse/zaptest"
 	"github.com/gorilla/mux"
 	"github.com/stretchr/testify/assert"
+	"go.mongodb.org/atlas/mongodbatlas"
 	corev1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	dbaas "github.com/mongodb/mongodb-atlas-kubernetes/pkg/api/dbaas"
+	v1 "github.com/mongodb/mongodb-atlas-kubernetes/pkg/api/v1"
+	status "github.com/mongodb/mongodb-atlas-kubernetes/pkg/api/v1/status"
```
Comment (tchughesiv) [inline pkg/controller/atlasinstance/atlasinstance_test.go:36]: are changes to this file necessary?

File: pkg/controller/atlasinventory/inventory_test.go [hunk 4]
```diff
@@ -37,9 +37,9 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/reconcile"
 
 	dbaasv1alpha1 "github.com/RHEcosystemAppEng/dbaas-operator/api/v1alpha1"
-	dbaas "github.com/mongodb/mongodb-atlas-kubernetes/pkg/api/dbaas"
 	"go.mongodb.org/atlas/mongodbatlas"
 
+	"github.com/mongodb/mongodb-atlas-kubernetes/pkg/api/dbaas"
```
Comment (tchughesiv) [inline pkg/controller/atlasinventory/inventory_test.go:42]: are changes to this file necessary?

**Review by tchughesiv — CHANGES_REQUESTED** [review]

File: go.mod (hunk 1 above)
Comment (jianrongzhang89) [reply go.mod]: Good catch. I was able to delete this line.

File: pkg/controller/atlasinventory/inventory_test.go (hunk 4 above)
Comment (jianrongzhang89) [reply pkg/controller/atlasinventory/inventory_test.go:42]: Yes, to pass the lint checking issues.

File: go.sum (hunk 2 above)
Comment (jianrongzhang89) [reply go.sum]: Cleaned it up by rerunning go mod tidy.

File: pkg/controller/atlasinstance/atlasinstance_test.go (hunk 3 above)
Comment (jianrongzhang89) [reply pkg/controller/atlasinstance/atlasinstance_test.go]: Backed out the format changes and will submit another PR for it.

File: pkg/controller/atlasinventory/inventory_test.go (hunk 4 above)
Comment (jianrongzhang89) [reply pkg/controller/atlasinventory/inventory_test.go]: Backed out the change for the formatting issue.

**Review by tchughesiv — APPROVED** [review]
lgtm

**Pull request merged**